### ISSUE TITLE
Getting an unexpected EOF when trying to download 'bert-large-uncased-whole-word-masking-finetuned-squad' model.

### ISSUE THREAD

**Issue by agrawalarpan** [huggingface/transformers#1303]
## 🐛 Bug

Model I am using (Bert, XLNet....): Bert

Language I am using the model on (English, Chinese....): English

The problem arise when using:
* [ ] the official example scripts: (give details): BertForQuestionAnswering
* [ ] my own modified scripts: (give details)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name): SQuaD
* [ ] my own task or dataset: (give details)

## To Reproduce

1. from pytorch_transformers import BertForQuestionAnswering
2. model = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad')

## Environment

* OS:
* Python version:
* PyTorch version:
* PyTorch Transformers version (or branch):
* Using GPU ?
* Distributed of parallel setup ?
* Any other relevant information:

**Comment by TheEdoardo93** [comment]
In my environment, it works as expected!

_Environment_:

- **Python**: 3.6.9
- **O.S.** : Linux-4.15.0-70-generic-x86_64-with-debian-buster-sid
- **Transformers**: 2.1.1 (installed from source with `pip install git+https://github.com/huggingface/transformers.git`)
- **Torch**: 1.3.1

_Example code_:
```
>>> import transformers
>>> from transformers import BertForQuestionAnswering
>>> model = BertForQuestionAnswering.from_pretrained(''bert-large-uncased-whole-word-masking-finetuned-squad'')
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 341/341 [00:00<00:00, 152821.63B/s]
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1340675298/1340675298 [02:04<00:00, 10793596.06B/s]
>>> ...
```

> ##  Bug
> Model I am using (Bert, XLNet....): Bert
> 
> Language I am using the model on (English, Chinese....): English
> 
> The problem arise when using:
> 
> * [ ]  the official example scripts: (give details): BertForQuestionAnswering
> * [ ]  my own modified scripts: (give details)
> 
> The tasks I am working on is:
> 
> * [ ]  an official GLUE/SQUaD task: (give the name): SQuaD
> * [ ]  my own task or dataset: (give details)
> 
> ## To Reproduce
> Steps to reproduce the behavior:
> 
> 1. from pytorch_transformers import BertForQuestionAnswering
> 2. model = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad')
> 
> ## Expected behavior
> ## Environment
> * OS:
> * Python version:
> * PyTorch version:
> * PyTorch Transformers version (or branch):
> * Using GPU ?
> * Distributed of parallel setup ?
> * Any other relevant information:
> 
> ## Additional context

**Comment by LysandreJik** [comment]
This is usually because of 
- a network error or
- not enough space on the disk

while downloading the file. To make sure it isn't the first, you can try running the `from_pretrained` method with the `force_download` option set to `True`:

```py
model = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad', force_download=True)
```
